pkg/azdiskschedulerextender/azdiskschedulerextender.go

```diff
@@ -1,3 +1,5 @@
+// +build azurediskv2
```

Comment:
I think we need to add this tag to all the files in pkg/azdiskschedulerextender and specifically to pkg/azdiskschedulerextender/main.go because it has a dependency on this file.

Reply:
Added